conda_build/utils.py

```diff
@@ -15,6 +15,8 @@
 
 from conda_build import external
 
+import logging
+logger = logging.getLogger('conda-build')
```

Comment:
I actually use `logging.getLogger(__name__)` too.  I thought that this was a convention that was used in `conda`, but I was mixing that up with some other project and overall just not following the correct convention.  I'll rework this to use __name__ in modules that now use the logging framework